modules/serverless-creating-broker.adoc

```diff
@@ -3,41 +3,144 @@
+----
 
-.Procedure
-. Create the `ServiceAccount` objects:
+.. Create the Service by applying the YAML file:
 +
+
+[source,terminal]
 ----
-$ oc -n default create serviceaccount eventing-broker-ingress
-$ oc -n default create serviceaccount eventing-broker-filter
+$ oc apply -f <FILENAME>
 ----
 
-. Give those objects RBAC permissions:
+. Optional. Create an event source that emits events to the `default` Broker. The following example shows how you can create a basic PingSource.
+.. Copy the following into a YAML file:
 +
+
+[source,yml]
 ----
-$ oc -n default create rolebinding eventing-broker-ingress \
-  --clusterrole=eventing-broker-ingress \
-  --serviceaccount=default:eventing-broker-ingress
-$ oc -n default create rolebinding eventing-broker-filter \
-  --clusterrole=eventing-broker-filter \
-  --serviceaccount=default:eventing-broker-filter
+apiVersion: sources.knative.dev/v1alpha2
+kind: PingSource
+metadata:
+  name: ping-source
+spec:
+  schedule: "*/1 * * * *"
+  jsonData: '{"message": "Hello serverless!"}'
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1beta1
+      kind: Broker
+      name: default
 ----
 
-. Create the broker:
 +
+..  Create the event source by applying the YAML file:
+
++
+[source,terminal]
+----
+$ oc apply -f <FILENAME>
+----
+
+. Create the Broker. You can do this by creating a Trigger object that uses the `knative-eventing-injection: enabled` annotation.
++
+[IMPORTANT]
+====
+If you create a Broker using the `knative-eventing-injection: enabled` annotation, you cannot delete this Broker without cluster administrator permissions.
+If you delete the Broker without having a cluster administrator remove this annotation first, the Broker will be created again after deletion.
+====
+
+.. Copy the following into a YAML file:
++
+
+[source,yml]
 ----
-cat << EOF | oc apply -f -
 apiVersion: eventing.knative.dev/v1beta1
-kind: Broker
+kind: Trigger
 metadata:
-  namespace: default
-  name: default <1>
-EOF
+  annotations:
+    knative-eventing-injection: enabled
+  name: example-trigger <1>
+spec:
+  broker: default
+  subscriber: <2>
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+----
+
++
+<1> You can specify any valid name for the Trigger.
+<2> The subscriber configuration is used to specify the event sink. In this example, it uses the Service that was created in step 1 of this procedure.
+
++
+..  Create the Broker by applying the YAML file:
++
+
+[source,terminal]
+----
+$ oc apply -f <FILENAME>
+----
+
+.Verification steps
+
+You can verify that events were sent to the event sink by looking at the sink pod's logs.
+
+By default, Knative Services terminate their pods if no traffic is received within a 60 second period.
+The example shown in this guide creates a PingSource that sends a message every 2 minutes, so each message should be observed in a newly created pod.
```

Comment:
s/guide/procedure
I'd rewrite to avoid "should."

>! should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as appropriate.

Reply:
As per comment above, I'd like to follow this up in a separate PR since it may take longer and the docs already exist here - I'd rather not block adding new information for a rewrite that can be done async :slightly_smiling_face: wdyt?